text/0032-access_command.md

```diff
@@ -0,0 +1,224 @@
+
+```
+$ qri access hidden --help
+hidden "unlists" a dataset. A hidden dataset will not show up when browsing 
+dataset lists or feeds. Others can still access a hidden dataset if they know
+its name.
+
+hidden datasets cannot be pushed to the registry, but can be pushed to peers 
+that accept hidden datasets from you.
+
+Usage:
+  qri access hidden [DATASET] [DATASET...] [flags]
+
+Examples:
+  # unpublish a dataset, removing it from any lists or feeds
+  $ qri access unpublish me/dataset
+
+  # Publish a few datasets:
+  $ qri access unpublish me/dataset me/other_dataset
+ 
+Flags:
+  -r, --remove     issue "remove all" commands to any published remotes
+```
+
+
+`visible` & `hidden` on their own may not seem like they warrant being moved into a subcommand. To understand the justification for the `access`, this is how `access` _might_ look one day, after we've landed encryption and role-based access control (RBAC) for datasets:
+
+```
+$ qri access --help
+Access controls visibility and permissions for datasets.
+
+Usage:
+  qri access [command]
+
+Available Commands:
+  info           show access control details for a dataset
+  visible        list dataset for others to see and pull
+  hidden         unlist dataset, removing it from public visibility
+  encrypt        restrict access to a dataset with a secret key
+  decrypt        remove access restrictions from a dataset
+  allow          give permissions to a peer for an encrypted dataset
+  deny           remove peer access permissions
+```
+
+### `qri access info`
+```
+$ qri access info --help
+Info describes permissions for a dataset.
+
+Usage
+  qri access info [DATASET] [DATASET...]
+```
+
+An example output for a published dataset:
+```
+$ qri access info me/dataset
+can-edit:  true
+visible:   true
+
+You can edit this dataset.
+
+This dataset is visible. Other users may see it when you're online, and if
+pushed to the registry it will show up in feeds for other users to browse and
+pull.
+```
+
+An example for a dataset you _don't_ own:
+```
+$ qri access info ca-state-parks/park-features
+can-edit:  false
+visible:   true
+
+You cannot edit this dataset, attempting to save to its history will create a 
+fork.
+
+This dataset is visible. Other users may see it when you're online, and if
+pushed to the registry it will show up in feeds for other users to browse and
+pull.
+```
+
+`$ qri access info me/dataset` for now will only but it will be _very_ helpful for viewing dataset permsissions that control both who can access a dataset and what they can do with it.
+
+### Visible on registry push
+Pushing unencrypted datasets† to the registry _requires_ a dataset be visible. To keep the old behaviour of a one-liner publish, pushing to the registry will automatically run `$ qri access visible` on the user's behalf. The push command will present this as user feedback:
```

Comment:
Discussed earlier on video, I agree that we should change this language to say "Pushing unencrypted datasets† to the registry defaults to making that dataset become visible."

We should support pushing hidden datasets, but it should require additional flags from the user, making it an opt-in